### PR TITLE
Edit password

### LINKED ISSUE #21: User Story 21, User Can Edit their Password
As a registered user
When I visit my profile page
I see a link to edit my password
When I click on the link to edit my password
I see a form with fields for a new password, and a new password confirmation
When I fill in the same password in both fields
And I submit the form
Then I am returned to my profile page
And I see a flash message telling me that my password is updated

### PR DESCRIPTION
**Pull request by JoseLopez235**
PR Contains:
- Adds edit page for passwords
- Updates password to new password 
- All Passing Test

closes #21

### CODE REVIEW

File: spec/features/users/edit_password_spec.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+# User Story 21, User Can Edit their Password
```
Comment (stillsheryl) [inline spec/features/users/edit_password_spec.rb]: Might want to take out the comments here!

File: config/routes.rb [hunk 2]
```diff
@@ -46,7 +46,8 @@
   get "/profile", to: "users#show"
   get "/profile/edit", to: "users#edit"
   patch "/profile/edit", to: "users#update"
-
+  get "/profile/edit/password", to: "users#edit_password"
```
Comment (stillsheryl) [inline config/routes.rb:49]: I wonder if we should consider having a PasswordsController?

Something like the update password in this [article](https://www.sitepoint.com/handle-password-and-email-changes-in-your-rails-api/).  Maybe something to consider for a future refactor!

**Review by KieraAllen — APPROVED** [review]
Removed User Story 21 comments. Looks great!

File: spec/features/users/edit_password_spec.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      visit '/login'
+
+      fill_in :email, with: @kiera.email
+      fill_in :password, with: @kiera.password
+
+      click_button 'Login'
+    end
+
+    it 'I see a link to edit my password' do
+      visit '/profile'
```
Comment (c-bartell) [inline spec/features/users/edit_password_spec.rb:17]: Great the use of before(:each)!

File: config/routes.rb (hunk 2 above)
Comment (c-bartell) [reply config/routes.rb:49]: I think this is a good idea, to stay RESTful.

**Review by c-bartell — APPROVED** [review]

**Pull request merged**